plugin/al/usdtransaction/AL/usd/transaction/tests/CMakeLists.txt

```diff
@@ -41,17 +43,55 @@ add_test(
     ${USDTRANSACTION_TEST_EXECUTABLE_NAME}
 )
 
+if(IS_WINDOWS)
+  set(path
+      "${CMAKE_INSTALL_PREFIX}/lib/gtest"
+      "${CMAKE_INSTALL_PREFIX}/plugin/al/lib"
+      "$ENV{PATH}"
+  )
+  string(REPLACE ";" "\;" path "${path}")
+
+  set_property(TEST ${USDTRANSACTION_TEST_NAME} APPEND PROPERTY ENVIRONMENT 
+      "PATH=${path}"
+  )
+endif()
+
 add_test(
   NAME ${USDTRANSACTION_PYTHON_TEST_NAME}
   COMMAND
     python -m unittest discover -s ${CMAKE_CURRENT_SOURCE_DIR}
 )
 
-set_tests_properties(${USDTRANSACTION_PYTHON_TEST_NAME}
-  PROPERTIES
-    ENVIRONMENT PYTHONPATH=${CMAKE_BINARY_DIR}/${INSTALL_DIR_SUFFIX}/usdtransaction:${CMAKE_BINARY_DIR}/src:$ENV{PYTHONPATH} 
+# set the PYTHONPATH and PATH for the python unittests
+set(pythonPath
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/lib/python"
+    "$ENV{PYTHONPATH}"
+)
+
+set(path 
+    "${CMAKE_INSTALL_PREFIX}/plugin/al/lib"
+    "$ENV{PATH}"
 )
 
+if(IS_WINDOWS)
+  list(APPEND path "${CMAKE_INSTALL_PREFIX}/lib/gtest")
+endif()
+
+if(IS_WINDOWS)
+    string(REPLACE ";" "\;" pythonPath "${pythonPath}")
+    string(REPLACE ";" "\;" path "${path}")
+else()
+    separate_arguments(pythonPath NATIVE_COMMAND "${pythonPath}")
+    separate_arguments(path NATIVE_COMMAND "${path}")
+
+    string(REPLACE "\;" ":" pythonPath "${pythonPath}")
+    string(REPLACE "\;" ":" path "${path}")
+endif()
+
+set_property(TEST ${USDTRANSACTION_PYTHON_TEST_NAME} APPEND PROPERTY ENVIRONMENT 
+    "PYTHONPATH=${pythonPath}"
+    "PATH=${path}"
+)
```

Comment:
All the bash scripts for setting and running python unit tests are now removed for the sake of portability.

CMake `PROPERTY ENVIRONMENT` can be used instead to set ENV variables during the unit tests run.